README.md

```diff
@@ -276,11 +276,12 @@ The below email providers are private, end-to-end encrypted (E2EE) and reasonabl
 
 | Provider | Description |
 | --- | --- |
-**[ProtonMail](https://protonmail.com/)** | An open-source, end-to-end encrypted anonymous email service. ProtonMail has a modern easy-to-use and customizable UI, as well as fast, secure native mobile apps. ProtonMail has all the features that you'd expect from a modern email service and is based on simplicity without sacrificing security. It has a free plan or a premium option for using custom domains (starting at $5/month). ProtonMail requires no personally identifiable information for signup, they have a [.onion](https://protonirockerxow.onion) server, for access via Tor, and they accept anonymous payment: BTC and cash (as well as the normal credit card and PayPal).
-**[Tutanota](https://tutanota.com/)** | Free and open source email service based in Germany. It has a basic intuitive UI, secure native mobile apps, anonymous signup, and a .onion site. Tutonota has a full-featured free plan or a premium subscription for businesses allowing for custom domains ($12/ month).<br>Tutanota [does not use OpenPGP](https://tutanota.com/blog/posts/differences-email-encryption/) like most encrypted mail providers, instead they use a standardized, hybrid method consisting of a symmetrical and an asymmetrical algorithm (with 128 bit AES, and 2048 bit RSA). This causes compatibility issues when communicating with contacts using PGP. But it does allow them to encrypt much more of the header data (body, attachments, subject lines, and sender names etc) which PGP mail providers cannot do
+**[Forward Email](https://forwardemail.net)** | 100% open-source and privacy-focused encrypted email service that supports [SMTP](https://forwardemail.net/faq#do-you-support-sending-email-with-smtp), [IMAP](https://forwardemail.net/faq#do-you-support-receiving-email-with-imap), [POP3](https://forwardemail.net/faq#do-you-support-pop3), [email API](https://forwardemail.net/faq#do-you-support-sending-email-with-api), [email webhooks](https://forwardemail.net/en/faq#do-you-support-webhooks), [regex](), unlimited aliases, unlimited domains, [OpenPGP/WKD](https://forwardemail.net/faq#do-you-support-openpgpmime-end-to-end-encryption-e2ee-and-web-key-directory-wkd), and more.  [Paid plans start at $3/mo for an enhanced set of features](https://forwardemail.net/private-business-email?pricing=true).  It is compatible with Apple Mail, Gmail, Mozilla, Thunderbird, K-9 Mail, and any open-source email client.  Unlike other email providers, it is 100% open-source and uses [individually encrypted SQLite mailboxes](https://forwardemail.net/blog/docs/best-quantum-safe-encrypted-email-service) to store your emails, where only you have the password.
```

Comment:
> `[regex]()`

Missing link. (nice feature btw, haven't seen this on other email forwarders!)


> `It is compatible with Apple Mail, Gmail, Mozilla, Thunderbird, K-9 Mail, and any open-source email client.`

Just say what protocols it works with, this is unecissary. 


> `Unlike other email providers`

Reads like an advert. Let's keep things objective.